library/src/test/java/hu/supercluster/overpasser/library/query/UsageExamplesTest.java

```diff
@@ -123,4 +123,43 @@ public void testMultipleFilterQueriesWithGlobalBoundingBoxSearch() throws Except
 
         assertEquals(expected, result);
     }
+
+    @Test
+    public void testMultipleFilterQueriesAround() throws Exception {
+        String result = new OverpassQuery()
+                .format(JSON)
+                .timeout(30)
+                .filterQuery()
+                .node()
+                .amenity("parking")
+                .tagNot("access", "private")
+                .around(50.5)
+                .prepareNext()
+                .way()
+                .amenity("parking")
+                .tagNot("access", "private")
+                .around(
```

Comment:
I know Android Studio flattens these methods to the same indentation level, but please observe how other code is formatted. For greater readability, methods between `.filterQuery()` and `.endQuery()`, and `.node()` / `.way()` / etc. and `.prepareNext()` must be one level deeper regarding indentation. Please follow the style of the rest of the file.